variables.tf

```diff
@@ -294,21 +294,65 @@ variable "disk_public_network_access" {
   default     = false
 }
 
-variable "alerts_email_recipients" {
-  description = "List of e-mail recipients for alerts"
-  type        = list(string)
-  default     = []
+# APPI related variables
+
+variable "workspace_retention_in_days" {
```

Comment:
There are difrerent workspaces in azure, not only for log analytics.

Reply:
fixed